aggregations/db_tables/near_ecosystem_entities.py

```diff
@@ -31,13 +38,58 @@ def sql_select(self):
     @property
     def sql_insert(self):
         return """
-            INSERT INTO near_ecosystem_entities VALUES %s
+            INSERT INTO near_ecosystem_entities VALUES(%s %s %s %s %s %s %s %s %s %s %s, %s, %s, %s, %s, %s) 
             ON CONFLICT DO NOTHING
         """
 
     def collect(self, requested_timestamp: int):
-        with open("aggregations/csv/near_ecosystem_entities.csv") as csvFile:
-            read = csv.reader(csvFile, delimiter=",")
-            result = list(read)
-            csvFile.close()
-            return result
+        pass
+
+    def store(self, parameters: list):
+        url = "https://raw.githubusercontent.com/near/ecosystem/main/entities.json"
+        r = requests.get(url).text
+        data = []
+        data = json.loads(r)
+
+        sql_insert = """
+            INSERT INTO near_ecosystem_entities VALUES(%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s) 
+            ON CONFLICT DO NOTHING
+        """
+
+        with self.analytics_connection.cursor() as analytics_cursor:
+            # Remove & update schema
+            try:
+                analytics_cursor.execute(self.sql_drop_table)
+                self.analytics_connection.commit()
+            except psycopg2.errors.UndefinedTable:
+                self.analytics_connection.rollback()
+            try:
+                analytics_cursor.execute(self.sql_create_table)
+                self.analytics_connection.commit()
+            except psycopg2.errors.DuplicateTable:
+                self.analytics_connection.rollback()
```

Comment:
you can use drop/create logic from the base class, no need to duplicate this, just invoke the corresponding methods